addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -369,7 +369,7 @@ def _import_attachments(self, invoice, tree):
 
         return attachments
 
-    def _import_partner(self, company_id, name, phone, email, vat, peppol_eas=False, peppol_endpoint=False, postal_address={}):
+    def _import_partner(self, *, company_id, name, phone, email, vat, peppol_eas=False, peppol_endpoint=False, postal_address={}, **kwargs):
```

Comment:
```suggestion
    def _import_partner(self, company_id, name, phone, email, vat, *, peppol_eas=False, peppol_endpoint=False, postal_address={}, **kwargs):
```

the 5 first params are identical in all overrides, and the definition shouldn't be updated for the uses, the uses should be adapted to the method definition instead (if needed).

Reply:
Actually, there is one override and its first argument is a `company_id` - in sale_edi_ubl. All calls are done using kwargs already (with the exception of that override).

Reply:
Updated.